corehq/apps/builds/tests/test_utils.py

```diff
@@ -0,0 +1,48 @@
+from django.test import SimpleTestCase
+from corehq.apps.builds.utils import _parse_version, is_out_of_date
+
+
+class TestVersionUtils(SimpleTestCase):
+    def test_parse_version(self):
+        test_cases = [
+            ('2.53.0', (2, 53, 0)),
+            ('2.53', (2, 53)),
+            ('1.0.0', (1, 0, 0)),
+            ('7', (7,)),
+            ('', None),
+            (None, None),
+            ('2.53.0.invalid', None),
+            ('not_a_version', None),
+        ]
+
+        for version, expected in test_cases:
+            with self.subTest(version=version, expected=expected):
+                result = _parse_version(version)
+                self.assertEqual(
+                    result,
+                    expected,
+                    f"Failed to parse '{version}'. Expected {expected}, got {result}"
+                )
```

Comment:
Test can be removed

Reply:
3016d9e0505c907532e77a036bec54e81afeb813